tests/integration/test_dynamodb.py

```diff
@@ -948,6 +948,34 @@ def test_transaction_write_items(self):
         # clean up
         dynamodb.delete_table(TableName=table_name)
 
+    def test_transaction_write_binary_data(self):
+        table_name = "test-ddb-table-%s" % short_uid()
+        dynamodb = aws_stack.create_external_boto_client("dynamodb")
+        dynamodb.create_table(
+            TableName=table_name,
+            KeySchema=[{"AttributeName": "id", "KeyType": "HASH"}],
+            AttributeDefinitions=[{"AttributeName": "id", "AttributeType": "S"}],
+            ProvisionedThroughput={"ReadCapacityUnits": 5, "WriteCapacityUnits": 5},
+            Tags=TEST_DDB_TAGS,
+        )
+        response = dynamodb.transact_write_items(
+            TransactItems=[
+                {
+                    "Put": {
+                        "TableName": table_name,
+                        "Item": {
+                            "id": {"S": "someUser"},
+                            "binaryData": {"B": b"foobar"},
+                        },
+                    }
+                }
+            ]
+        )
+        assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
```

Comment:
is there some other, more strong assert that we could do here? maybe get the item and check whether it was written correctly?

Reply:
Good point, added a more meaningful assertion!